core/constant_inc.php

```diff
@@ -577,7 +577,7 @@
 define( 'EMAIL_SHUTDOWN_FORCE', 2 );
 
 # Lengths - NOTE: these may represent hard-coded values in db schema and should not be changed.
-define( 'DB_FIELD_SIZE_USERNAME', 255 );
+define( 'DB_FIELD_SIZE_USERNAME', 191 );
```

Comment:
I would add a comment explaining why we came up with this magic number and if there is a point in the future where we can revisit.